lale/operators.py

```diff
@@ -1285,6 +1315,34 @@ def __init__(
     def _is_instantiated(self):
         return not inspect.isclass(self._impl)
 
+    def _get_trained_op_list(self):
```

Comment:
I find this name confusing, because it is not retrieving a list of operators. Instead, if I understand your code correctly, it is retrieving a list of attributes that are masked in trained Lale operators. How about renaming it to `_get_masked_attr_list` or something like that?